src/bin/pgcopydb/cli_dump.c

```diff
@@ -370,7 +380,7 @@ cli_dump_schema_section(CopyDBOptions *dumpDBoptions,
 		exit(EXIT_CODE_INTERNAL_ERROR);
 	}
 
-	if (!copydb_init_specs(&copySpecs, dumpDBoptions, DATA_SECTION_NONE))
+	if (!copydb_init_specs(&copySpecs, dumpDBoptions, DATA_SECTION_ALL))
```

Comment:
I'm not sure that we need DATA_SECTION_ALL here, but thinking about it, we might. After all we want to be able to implement the whole filtering.